tests/testsuite/clippy.rs

```diff
@@ -0,0 +1,39 @@
+use crate::support::{clippy_is_available, is_nightly, project};
+
+#[cargo_test]
+fn clippy() {
```

Comment:
I'd still like to see this test moved back to its original location.

Reply:
aah shit, forgot about that one.